packages/@sanity/base/src/presence/FieldPresence.tsx

```diff
@@ -41,7 +35,7 @@ function FieldPresenceWithOverlay(props: FieldPresenceProps) {
   const minWidth = -AVATAR_DISTANCE + (AVATAR_SIZE + AVATAR_DISTANCE) * props.maxAvatars
 
   return (
-    <div ref={ref} className={styles.root} style={{minWidth: minWidth, minHeight: AVATAR_SIZE}} />
+    <FlexBox justify="flex-end" ref={ref} style={{minWidth: minWidth, minHeight: AVATAR_SIZE}} />
```

Comment:
I find the naming here a bit confusing. Is `FlexBox` any different from `<Flex><Box>(...)</Box></Flex>`?

Reply:
Ah, this is just a misunderstanding 😄 
I was following one of the suggestions of the naming guidelines which suggested "[name]box" and I understand that in this situation it would be very confusing. It's unrelated. 

I'm going to be changing it to `FlexWrapper` :)